### PR TITLE
Fix Settings icon

### PR DESCRIPTION
**Pull request by Soitora**
Current: https://i.imgur.com/SpW7ErR.png
Fix: https://i.imgur.com/PI7ARBb.png

### CODE REVIEW

**Comment by SimonAlling** [comment]
Just wanted to say that I'll take care of this as soon as I have time. (Final two weeks of thesis right now.)

**Comment by Soitora** [comment]
No problem, I'm releasing a rework of ny theme soon too which much better code, just figuring some things out before release.

File: src/styles/stylesheet.scss [hunk 1]
```diff
@@ -39,7 +39,7 @@ textarea.#{getGlobal("CONFIG.CLASS.codeInput")}, .#{getGlobal("CONFIG.CLASS.code
 
 ##{getGlobal("CONFIG.ID.preferencesShortcut")} {
     background-image: url("#{getGlobal("SITE.URL_ICONS_GENERAL")}") !important; // to override Blargmode
-    background-position: -164.5px -95px;
+    background-position: -165px -94px;
```
Comment (SimonAlling) [inline src/styles/stylesheet.scss]: The height shouldn't be modified.

```suggestion
    background-position: -165px -95px;
```

Comment (Soitora) [reply src/styles/stylesheet.scss]: It looks much more centered on `-94px`, but whatever works for you.

**Comment by LemonIllusion** [comment]
-165px will not properly center the sprite. -164.5px is more centered, but the main issue is that the containing element is too wide for the icon, causing the nearby icons to show, and the width is also determined by the font, which sounds way too unpredictable to rely on. I suggest the following:
- Drop the NBSP from the `<span>` and set it `display: inline-block` to make the element the same height as the others (defined by `font-size`) but with no width.
- Use -166px for perfect horizontal centering
- -95px is the best approximation for vertical centering from what I could find, but this might be affected by available fonts and text renderer

**Comment by SimonAlling** [comment]
I have forgot to mention it, but I think @LemonIllusion's suggestion is good.

**Comment by Soitora** [comment]
Adjusted as per the suggestions from @LemonIllusion, also I added in-line styling for the `display: inline-block` if that's okay instead of a seperate `.scss` style.

Haven't tried if it works but I don't see why it shouldn't.

**Pull request merged**